app/models/game.rb

```diff
@@ -1,27 +1,15 @@
 class Game < ApplicationRecord
   has_many :turns
+  has_and_belongs_to_many :users
+  validates_associated :users
```

Comment:
This does not do what you think it does. Should look like
`validates :tags, length: { minimum: 1, maximum: 6 }`